tests/tracking/fluent/test_fluent.py

```diff
@@ -1257,7 +1257,7 @@ def test_log_input(tmp_path):
     df.to_csv(path)
```

Comment:
Can we add a test case for this to `test_rest_tracking.py`? REST tracking was where the issue was first discovered, so it'd be awesome to have some automated CI ensuring that it works :)

Reply:
We should add a test case that calls `log_input` :D. Given an `MlflowClient` instance, you can access the tracking URI via `client.tracking_uri`. You can then do `mlflow.set_tracking_uri(client.tracking_uri)` and proceed to call `mlflow.log_input()`

Reply:
Thank you!